public/API/API_GetUsersFollowingMe.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ *  API_GetUsersFollowingMe - returns list of the caller's follower users
+ *    o : offset - number of entries to skip (default: 0)
+ *    c : count - number of entries to return (default: 100, max: 500)
+ *  int         Count                       number of user records returned in the response
+ *  int         Total                       number of user records the user actually has overall
+ *  array       Results
+ *   object      [value]
+ *    string     User                       username
+ *    int        Points                     number of hardcore points the user has earned
+ *    int        PointsSoftcore             number of softcore points the user has earned
+ *    boolean    AmIFollowing               whether the caller user follows the follower user back
+ */
+
+use App\Community\Enums\UserRelationship;
+use App\Models\User;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Validator;
+
+$input = Validator::validate(Arr::wrap(request()->query()), [
+    'o' => 'nullable|integer|min:0',
+    'c' => 'nullable|integer|min:1|max:500',
+]);
+
+$offset = $input['o'] ?? 0;
+$count = $input['c'] ?? 100;
+
+/** @var User $user */
+$user = Auth::user();
+
+$totalUsers = $user->followerUsers()
+    ->whereNull('Deleted')
+    ->count();
+
+$usersList = $user
+  ->followerUsers()
+  ->whereNull("Deleted")
+  ->with([
+      "inverseRelatedUsers" => fn ($q) => $q
+        ->select(sprintf("%s.ID", $user->getTable()), "related_user_id")
+        ->where("user_id", $user->id)
+        ->withPivot("Friendship"),
+  ])
+  ->orderBy("LastActivityID", "DESC")
```

Comment:
```suggestion
  ->orderByDesc("LastActivityID")
```

nit: `->orderByDesc()` is even more idiomatic. These two endpoints would be the only place we have `->orderBy("Foo", "DESC")` (desc in all-caps).

Reply:
Thank you for the suggestion!